### PR TITLE
chg: resolves #579

### LINKED ISSUE #579: Update cu_search for legacy express url variables
cu_path
https://github.com/CuBoulder/express_mono/blob/dev/modules/features/cu_search/cu_search.install#L19-L27

### PR DESCRIPTION
**Pull request by AlanBCole**

### CODE REVIEW

**Comment by AlanBCole** [comment]
Basically just switched cu_path with base_url.

**Comment by AlanBCole** [comment]
@cathysnider Ah typos!

File: modules/features/cu_search/cu_search.install [hunk 1]
```diff
@@ -4,7 +4,7 @@
  * Implements hook_install().
  */
 function cu_search_install() {
-  variable_set('google_appliance_collection', 'default_collection');
```
Comment (jwfuller) [inline modules/features/cu_search/cu_search.install:7]: I don't think this part is necessary

**Comment by AlanBCole** [comment]
OK. I'll remove this code.

**Pull request merged**